lib/runtime.js

```diff
@@ -3,17 +3,16 @@
 var Entry = require("./entry.js");
 var utils = require("./utils.js");
 
-exports.enable = function (Module) {
-  var Mp = Module.prototype;
-
-  if (typeof Mp.importSync !== "function" ||
-      typeof Mp.export !== "function") {
-    Mp.runModuleSetters = runModuleSetters;
-    Mp.importSync = moduleImport;
-    Mp.export = moduleExport;
+exports.enable = function (mod) {
```

Comment:
How would you feel about allowing a constructor function, too?
```js
exports.enable = function (mod) {
  if (typeof mod === "function") mod = mod.prototype;
  ...
};
```
With that option, a caller who knows what they're doing could set `Module.prototype.{export,importSync,runModuleSetters}` for the entire process, overriding individual Reify versions. It would also be backwards compatible.